src/Symfony/Bundle/FrameworkBundle/Controller/ControllerResolver.php

```diff
@@ -35,39 +34,19 @@ class ControllerResolver extends BaseControllerResolver
      */
     public function __construct(ContainerInterface $container, ControllerNameParser $parser, LoggerInterface $logger = null)
```

Comment:
Because the new class is available in the `HttpKernel` component, whereas this one is in the framework bundle, as well as the `ControllerNameParser`.
I'm fine with this class as is in the framework. I just think having a base ControllerResolver usable with psr-11 shipped with the component makes sense.